lib/convection/model/template/resource_property/aws_s3_replication_configuration_rule.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../resource_property'
+
+module Convection
+  module Model
+    class Template
+      class ResourceProperty
+        #Represents an {https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket-replicationconfiguration-rules.html}
```

Comment:
These comments will fail rubocop linting 👮 due to lack of a space after the hash in the comment.